core/rfb.js

```diff
@@ -2600,3 +2630,17 @@ RFB.encodingHandlers = {
         }
     }
 }
+
+RFB.cursors = {
+    dot: {
+        rgbaPixels: new Uint8Array(Base64.decode(
+            "/////wAAAP//////" +
+            "AAAA/wAAAAAAAAD/" +
+            "/////wAAAP//////"
```

Comment:
Nitpick, but do we really need to pull in Base64 just for this? It's just 3x3 pixels. The array can't be that large?